client.go

```diff
@@ -23,6 +23,16 @@ import (
 
 const DEFAULT_PAGE_LENGTH = 10
 const DEFAULT_MAX_DEPTH = 1
+const DEFAULT_BITBUCKET_API_BASE_URL = "https://api.bitbucket.org/2.0"
```

Comment:
Not sure if this is the best way to solve this. But it works and so thought I would put it in the PR so people with more experience than I can provide feedback :)